client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
+    });
+
```

Comment:
is the quotes thing a standard in React?
On 06/02/2015 06:00 PM, lifehackett wrote:
>
> In client/src/js/components/map.jsx 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8#discussion_r31580187>:
>
> > @@ -39,7 +39,17 @@ var Map = React.createClass({
> >        }
> >      ).addTo(map);
> >
> > -    L.geoJson(items).addTo(map);
> > +    var marker = new
> > +    L.Icon({
> > +      iconUrl: "images/marker-icon.png"
> > +    });
> > +
>
> Style note- line breaks and single quotes
>
> var  marker=  new  L.Icon({
>    iconUrl:  'images/marker-icon.png'
> });
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8/files#r31580187>.
>